apps/angular/src/app/features/anime/anime-details/anime-details.component.html

```diff
@@ -1,38 +1,56 @@
-<main class="anime-details" *ngIf="animeDetails$ | async as animeDetails">
-	<h1 class="anime-details-title">{{animeDetails.englishTitle ? animeDetails.englishTitle : animeDetails.japaneseTitle}}</h1>
+<main
+  *ngIf="animeDetails$ | async as animeDetails"
+  class="anime-details">
+	<a routerLink="/anime" mat-icon-button>
+		<mat-icon>arrow_back</mat-icon>
+	</a>
+	<a routerLink="edit" mat-icon-button>
+		<mat-icon>update</mat-icon>
+	</a>
+	<button 
+	  type="button"
+	  mat-icon-button
+	  (click)="openDeletionConfirmation(animeDetails.id)">
+		<mat-icon>delete</mat-icon>
+	</button>
+	<h1 class="anime-details__title typography typography_title">{{animeDetails.englishTitle ? animeDetails.englishTitle : animeDetails.japaneseTitle}}</h1>
```

Comment:
```suggestion
	<h1 class="anime-details__title typography typography_title">{{animeDetails.englishTitle != null ? animeDetails.englishTitle : animeDetails.japaneseTitle}}</h1>
```

Please avoid casting types. Also, what if the anime doesn't have a title in any of the languages? Maybe we should make a separate method for generating the title?